core/root.sol

```diff
@@ -30,14 +33,20 @@ contract RootZone {
     function ante(bytes32 hash) external payable {
         if (msg.value <= pile * 101 / 100) revert ErrPayment();
         if (block.timestamp >= term && pile != 0) revert ErrPending();
-        user.call{gas: 2300, value: pile}("");
+        back[user] += pile;
         pile = msg.value;
         user = msg.sender;
         dark = hash;
         term = block.timestamp + FREQ;
         emit Ante(pile);
     }
 
+    function withdraw() external {
```

Comment:
We should add a reentrancy test for withdraw() and probably set back[user] to zero before the refund